prow/cluster/deck_deployment.yaml

```diff
@@ -53,6 +53,9 @@ spec:
         - --job-config-path=/etc/job-config
         - --spyglass=true
         - --rerun-creates-job
+        - --github-token-path=/etc/github/oauth
```

Comment:
There won't be an `oauth` file here. We have the `github-oauth-config` secret mounted at the `/etc/github` path and used as the `--github-oauth-config-file`: https://github.com/kubernetes/test-infra/blob/6a85e1e9718d1d43aaa4b756826b36e8d22eff71/prow/cmd/deck/main.go#L149
You need to mount and reference the github token from the `oauth-token` secret instead.